src/vendor/protocol.d.ts

```diff
@@ -164,4 +164,24 @@ declare namespace lobby {
         /** Settings. */
         settings: LobbySetting[];
     }
+}
+
+declare namespace user {
+    /**
+     * Represents an user
+     */
+    interface User {
+        /** ID of the user */
+        id: number;
+        /** Name of the user */
+        username: string;
+        /** ID of the icon */
+        idIcon: number;
```

Comment:
Agreed about `idIcon` -> `iconId`. This should be done in numerous places, not just here.